app/Stores/EditStore.js

```diff
@@ -0,0 +1,195 @@
+import { EventEmitter } from 'events';
+import { Map } from 'immutable';
+import objectAssign from 'object-assign';
+
+import AppDispatcher from '../Services/AppDispatcher';
+
+import ReadQueries from 'admin-config/lib/Queries/ReadQueries';
+import WriteQueries from 'admin-config/lib/Queries/WriteQueries';
+import PromisesResolver from 'admin-config/lib/Utils/PromisesResolver';
+import DataStore from 'admin-config/lib/DataStore/DataStore';
+
+import RestWrapper from '../Services/RestWrapper';
+
+class EditStore extends EventEmitter {
```

Comment:
Structuration of stores should relate to the domain, not to the view (cf https://facebook.github.io/flux/docs/overview.html#stores). For ng-admin, there could be an `EntityStore` dealing with the logic for an entity, an `ApplicationStore` storing the app logic, and that's about all. In fact, the `DataStore` object could be extended to match the logic of a React _Store_. 

If the `Entity` store is too big, move part of its logic to services (which is already necessary in this class).